README.md

```diff
@@ -210,57 +210,16 @@ Normally end-users should not need to use builders, as they're primarily meant
 for maintainers - both scheme and template maintainers. These are tools used to
 build templates with all the currently submitted schemes.
 
-### Official Builder Repositories
+Spec changes will not be merged until there is consensus among maintainers and
```

Comment:
```suggestion
Spec changes will not be released until there is consensus among maintainers and
```

I thought we were doing releases rather than branches?

Reply:
Yes - I was aiming to keep these changes small, so I didn't include those changes in this PR.

Reply:
But your PR just added this line... why misstate the plan?  I think my one word correction clears it up (for now).

Reply:
It was copied from below - anyway, I made the change you requested before merging it in.

Reply:
Ohhhhhh